.taskcluster.yml

```diff
@@ -233,7 +233,9 @@ tasks:
                                 # Note: This task is built server side without the context or tooling that
                                 # exist in tree so we must hard code the hash
                                 image:
-                                    mozillareleases/taskgraph:decision-mobile-6607973bc60e32323a541861cc5856cd6a0f51ea9fd664ef7d43bca8df53db47@sha256:8c471aacc469ea8e7bb4846c16efe086f7350a5cc1df570cc6c86b22895a2456
+                                    type: indexed-image
+                                    path: public/image.tar.zst
+                                    namespace: taskgraph.cache.level-1.docker-images.v2.decision-mobile.hash.ec51f5a9e3d3cf4d21f7d7d56527a8e9e439bb5732dd01ad192ccdd332ac8789
```

Comment:
We shouldn't merge until we use a hash-pinned dockerhub image.
Requesting changes to make sure we don't merge without changing this.